src/test/scala/io/citrine/lolo/PerformanceTest.scala

```diff
@@ -47,8 +47,7 @@ class PerformanceTest {
     (timeTraining, timePredicting)
   }
 
-  @Test
-  @Category(Array(classOf[SlowTest]))
+  // @Category(Array(classOf[SlowTest]))
```

Comment:
I have often found it useful to drop a link to an issue opened to track the work alonside the commented out code.  It helps to raise awareness in sprint planning so that it isn't forgotten and gives people a place to discuss the change. 

YMMV.